app/2.0/nav.yaml

```diff
@@ -133,6 +133,15 @@
   - title: Polymer CLI
     path: /2.0/docs/tools/polymer-cli
     indent: True
+  - title: Polymer CLI commands
```

Comment:
The left nav looks pretty cluttered with all of the "Polymer CLIs" in there... Maybe we could shorten these three to:
CLI commands
Create an element project
Create an app project

(Just in the left nav. Full titles in the docs are a-ok.)